electron/worker/actions/generateProjectMetadata.ts

```diff
@@ -0,0 +1,37 @@
+import path from "path";
```

Comment:
Rename this file from  `generateProjectMetadata.ts` to `genProjectMetadata.ts`, to match convention with `genCode.ts`